spec/pragma.dd

```diff
@@ -204,9 +204,8 @@ pragma(linkerDirective, "/FAILIFMISMATCH:_ITERATOR_DEBUG_LEVEL=2");
 -----------------
 
     $(IMPLEMENTATION_DEFINED
-    $(P The string literal specifies a linker directive to be embedded in the generated object file.)
-
-    $(P Linker directives are only supported for MS-COFF output.)
+    The string literal specifies a linker directive to be embedded in the generated object file.
```

Comment:
I think that the sentence needs to be wrppaed in a `($P )` . This might be the cause of the documentation tester.

Reply:
`$(IMPLEMENTATION_DEFINED )` is used without `$(P )` in many places in the spec.